docs/user-guide/aws-examples/eks.rst

```diff
@@ -0,0 +1,229 @@
+======================================
+Running NeMo Curator on AWS EKS
+======================================
+
+--------------------------------------
+Background
+--------------------------------------
+AWS EKS is a fully managed service that makes it easier to run Kubernetes on AWS without needing to install, operate, and maintain your own Kubernetes control plane.
+
+Running NeMo Curator on AWS EKS offers streamlined Kubernetes management integrated with AWS services like CloudWatch for enhanced monitoring and logging, and native auto-scaling capabilities.
+
+For more details, refer to `EKS documentation <https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html>`__
+
+This guide covers all essential prerequisites. It includes an example demonstrating how to create an EFS storage class and offers step-by-step instructions for setting up an EFS Persistent Volume Claim to dynamically provisioning Kubernetes Persistent Volume. Furthermore, it outlines the required steps to deploy a Dask cluster and delves into utilizing the Kubernetes Python client library to assign NeMo-Curator tasks to the Dask scheduler.
```

Comment:
Nit: "assign NeMo-Curator tasks to the Dask scheduler" -> "run NeMo Curator scripts"